graceful/service_test.go

```diff
@@ -161,7 +161,9 @@ func ensurePidFileProcessKilled(t *testing.T) {
 		process, err := os.FindProcess(pid)
 		require.NoError(t, err)
 		err = process.Kill()
-		require.NoError(t, err)
+		if err != nil && !strings.Contains(err.Error(), "already finished") {
```

Comment:
Have you checked with @Soulou if this is normal ? This seems weird that the process has already exited.

Reply:
Hmm yes I don't really get this =\